scripts/setup_linux.sh

```diff
@@ -194,6 +194,9 @@ fi
 . "${CONDA_SH}"
 conda update -q -y conda
 
+# We need a MuJoCo key to import mujoco_py
+cp ${_arg_mjkey} "${HOME}"/.mujoco/mjkey.txt
```

Comment:
I think we need failsafes in case:
1. the user doesn't present a mujoco key
2. the mujoco key is not a text file
3. (if possible) the mujoco key is not valid, but we may not be able to check that before installing mujoco

Reply:
1. That's already checked above
2. `$(file --mime-type "${_arg_mjkey}") == "text/plain")` or similar
3. Not possible.

Reply:
Great!! And I don't think 3 is possible outside the mujoco install process

Reply:
`test -f` checks that it's a file, but does not check that it's text.